index.js

```diff
@@ -221,51 +221,66 @@ const formatResults = (
 
 // Processes records and either extracts Typed Values into an array, or
 // object with named column labels
-const formatRecords = (recs,columns) => {
+const formatRecords = (recs, columns) => {
```

Comment:
Refactored this a bit to be more readable and do fewer object lookups and loops. From testing, execution time of this version is overall faster than the existing implementation, with occasionally being slower. 

I can post the logs for comparison in case anyone is interested.